dockerfiles/centos/Dockerfile

```diff
@@ -0,0 +1,48 @@
+# ------------------------------------------------------------------------
+#
+# Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+#
+# This software is the property of WSO2 Inc. and its suppliers, if any.
+# Dissemination of any information or reproduction of any material contained
+# herein is strictly forbidden, unless permitted by WSO2 in accordance with
+# the WSO2 Commercial License available at http://wso2.com/licenses.
+
+# For specific language governing the permissions and limitations under this
+# license, please see the license as well as any agreement you’ve entered into
+# with WSO2 governing the purchase of this software and any associated services.
+# ------------------------------------------------------------------------------
+
+# set base Docker image to centos7.7.1908 CentOS Docker image
+FROM centos:centos7.7.1908
+LABEL maintainer="WSO2 Docker Maintainers <dev@wso2.org>"
+
+# install required packages
+RUN yum -y update && yum install -y rpm-build sudo git unzip wget
```

Comment:
@RidmiR let us go with multi lines when defining package names and other best practices. Please see the format we have used in official product Docker source repositories (e.g. [APIM](https://github.com/wso2/docker-apim/blob/v3.1.0.3/dockerfiles/centos/apim/Dockerfile))

Reply:
@chirangaalwis Intially I wrote the docker file with multiple lines and it throws Warning. Show in Below

 **_[WARNING]: Empty continuation line found in:
    RUN yum -y update && yum install -y rpm-build sudo git unzip wget RUN echo "net.ipv4.ip_local_port_range=15000 61000" >> /etc/sysctl.conf && echo "net.ipv4.tcp_fin_timeout=30" >> /etc/sysctl.conf && echo "*	soft	nofile	65535" >> /etc/security/limits.conf && echo "*	hard	nofile	65535" >> /etc/security/limits.conf && echo "*	soft	nproc	65535" >> /etc/security/limits.conf && echo "*	hard	nproc	65535" >> /etc/security/limits.conf_**

As a fix, I remove and make it one line. Do I need to revert it?

Reply:
Fixed by changing into multiple lines